builder/builder-next/worker/worker.go

```diff
@@ -50,7 +50,7 @@ import (
 )
 
 func init() {
-	version.Version = "v0.11.7-0.20230525183624-798ad6b0ce9f"
+	version.Version = "v0.11.6+0a15675913b7"
```

Comment:
Use this pattern to match what we are going to have in 25+: https://github.com/moby/moby/blob/b1ead679f370652003a724d50ede27586cbe3d85/internal/mod/mod.go#L48

Reply:
Maybe we should add a link to that code for reference 🤔 

(but fine for a follow-up)

Of course we just need a new tag to not have to deal with this at all 😉

Reply:
Yes would like a tag too but no functional change as you can see in vendoring. Will look for tagging this week if we don't have anything to backport to this branch.